source/gloperate/include/gloperate/pipeline/Pipeline.h

```diff
@@ -91,38 +91,31 @@ class GLOPERATE_API Pipeline : public Stage
     *
     *  @param[in] stage
     *    Stage (must NOT be null!)
-    *  @param[in] ownership
-    *    Property ownership
     */
-    void addStage(Stage * stage, cppexpose::PropertyOwnership ownership = cppexpose::PropertyOwnership::Parent);
+    void addStage(Stage * stage);
 
     /**
     *  @brief
-    *    Remove stage
+    *    Add stage
     *
     *  @param[in] stage
     *    Stage (must NOT be null!)
-    *
-    *  @return
-    *    'true' if the stage was removed, else 'false'
-    *
-    *  If the stage is not part of the pipeline, nothing happens
     */
-    bool removeStage(Stage * stage);
+    void addStage(std::unique_ptr<Stage> && stage);
 
     /**
     *  @brief
-    *    Remove and destroy stage
+    *    Remove stage
     *
     *  @param[in] stage
     *    Stage (must NOT be null!)
     *
     *  @return
-    *    'true' if the stage was removed and destroyed, else 'false'
+    *    'true' if the stage was removed, else 'false'
     *
     *  If the stage is not part of the pipeline, nothing happens
     */
-    bool destroyStage(Stage * stage);
+    bool removeStage(Stage * stage);
```

Comment:
This may be a crucial change. @sbusch42 does this correspond to the cppexpose interfaces and semantics?